next.config.js

```diff
@@ -15,4 +15,14 @@ const nextConfig = {
   },
 };
 
-module.exports = nextConfig;
+module.exports = {
+  nextConfig,
+  images: {
+    remotePatterns: [
+      {
+        protocol: 'https',
+        hostname: '**.org',
```

Comment:
It's not a prudent strategy to forgo on security by allowing every subdomain of `.org`. Allow only that which is actually accessed by your app.